db/seeds.rb

```diff
@@ -3,3 +3,9 @@
                           { name: 'ce_registry',
                             backup_item: 'ce-registry-test',
                             default: true }])
+
+admin = Admin.create(name: 'ce_admin')
```

Comment:
I think it's better to use `find_or_create_by` on these, to avoid duplication errors in case we need to run again, for whatever reason (for example when someone adds something new). 

Also I think they were using the `cer` (credential engine registry) prefix for everything, unless it changed again, it's a good idea to keep consistency.